packages/@sanity/form-builder/src/sanity/inputs/reference/ReferencePreview.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+
+import {ObjectSchemaType} from '@sanity/types'
+import {getDraftId} from '@sanity/base/_internal'
+import {Box, Flex, Inline, Label, Text, Tooltip, useRootTheme} from '@sanity/ui'
+import {EditIcon, PublishIcon} from '@sanity/icons'
+import Preview from '../../../Preview'
+import {ReferenceInfo} from '../../../inputs/ReferenceInput/types'
+import {TimeAgo} from '../../../inputs/ReferenceInput/utils/TimeAgo'
+import {TextWithTone} from './TextWithTone'
+
+/**
+ * Used to preview a referenced type
+ * Takes the reference type as props
+ * @param props
+ * @constructor
+ */
+export function ReferencePreview(props: {
+  referenceInfo: ReferenceInfo
+  refType: ObjectSchemaType
+  layout: string
+  showTypeLabel: boolean
+
+  // this provides us with a workaround for an issue with css modules (https://github.com/thysultan/stylis.js/issues/272)
+  // the workaround is to write a `data-selected` prop on the <TextWithTone> component and use that instead of not()
```

Comment:
should this use the `PreviewComponentType` somehow? i see this comment is duplicated in two places